### PR TITLE
Fibonacci

### PR DESCRIPTION
**Pull request by Lukasz-334**
fibonacci test.

### CODE REVIEW

**Comment by ziobron** [comment]
✅ CI Passed - Task: fibonacci
🏅 Granted 6 XP for @Lukasz-334

**Comment by ziobron** [comment]
⏱ 3 XP extra for @Lukasz-334 for on-time delivery

File: homework/fibonacci/fibonacci.hpp [hunk 1]
```diff
@@ -1,11 +1,32 @@
...
+   }
+   
 }
 
 int fibonacci_recursive(int sequence) {
-    // TODO: Your implementation goes here
-    return 0;
+    if (sequence == 0)return 0;
+    else if ((sequence ==1) || (sequence==2))return 1;
+    else return fibonacci_recursive(sequence -1)+fibonacci_recursive(sequence -2);
```
Comment (gyrydymptum) [inline homework/fibonacci/fibonacci.hpp:31]: else before return is not needed

**Comment by ziobron** [comment]
Thanks @gyrydymptum for Code Review.
🏅 1 XP granted

**Pull request closed without merging**